### PR TITLE
fixed css problem

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by LauraCourquin — DISMISSED** [review]
It's good for me ! :)

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

**Comment by LoicBrassart** [comment]
Plz solve conflicts

**Review by LauraCourquin — DISMISSED** [review]
good for me

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix Conflict

**Review by LauraCourquin — APPROVED** [review]
It's good for me !

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/components/Burger/index.jsx [hunk 1]
```diff
@@ -13,16 +14,29 @@ function Burger() {
...
+          </Link>
+          <Link to="/agenda">
+            <li className="picture">Mon agenda</li>
+          </Link>
+          <Link to="/about">
+            <li className="picture">A propos</li>
+          </Link>
+          <Link to="/logout">
+            <li className="picture">Déconnexion</li>
+          </Link>
```
Comment (LoicBrassart) [inline frontend/src/components/Burger/index.jsx:34]: Not a blocking one, but direct children of `<ul>` should be `<li>`, not `<Link>`

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**